src/components/node-list/node-list-tree.js

```diff
@@ -118,6 +127,23 @@ const TreeListProvider = ({
     }
   };
 
+  const onItemExpandToggle = (event, expandedItemIds) => {
```

Comment:
the `event` is not used here and can be taken out.

Reply:
With it being the first argument I'm not sure if it can be? Can you show how you mean?